src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1424,42 +1584,42 @@ private void hitBlockWithToolInHand(@Nullable final BlockPos blockPos, final boo
 
         this.swingArm(this.getActiveHand());
 
-        final IBlockState blockState = worldObj.getBlockState(blockPos);
+        final IBlockState blockState = world.getBlockState(blockPos);
         final Block block = blockState.getBlock();
         if (breakBlock)
         {
-            if (!worldObj.isRemote)
+            if (!world.isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
-                  new BlockParticleEffectMessage(blockPos, worldObj.getBlockState(blockPos), BlockParticleEffectMessage.BREAK_BLOCK),
-                  new NetworkRegistry.TargetPoint(worldObj.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_SOUND_RANGE));
+                  new BlockParticleEffectMessage(blockPos, world.getBlockState(blockPos), BlockParticleEffectMessage.BREAK_BLOCK),
+                  new NetworkRegistry.TargetPoint(world.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_SOUND_RANGE));
             }
-            worldObj.playSound(null,
+            world.playSound(null,
               blockPos,
-              block.getSoundType(blockState, worldObj, blockPos, this).getBreakSound(),
+              block.getSoundType(blockState, world, blockPos, this).getBreakSound(),
               SoundCategory.BLOCKS,
-              block.getSoundType(blockState, worldObj, blockPos, this).getVolume(),
-              block.getSoundType(blockState, worldObj, blockPos, this).getPitch());
-            worldObj.setBlockToAir(blockPos);
+              block.getSoundType(blockState, world, blockPos, this).getVolume(),
+              block.getSoundType(blockState, world, blockPos, this).getPitch());
+            world.setBlockToAir(blockPos);
 
             damageItemInHand(1);
         }
         else
         {
             //todo: might remove this
-            if (!worldObj.isRemote)
+            if (!world.isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
                   //todo: correct side
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)